### PR TITLE
[onert] Add nnfw_api tests for ResizeBilinear

### PR DESCRIPTION
**Pull request by ragmani**
For #4053, #4115

This commid adds nnfw_api tests for ResizeBilinear.

Signed-off-by: ragmani <ragmani0216@gmail.com>

### CODE REVIEW

File: tests/nnfw_api/src/one_op_tests/ResizeBilinear.cc [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  tcd.addInput(std::vector<int32_t>{3, 3});
+  tcd.addInput(std::vector<float>{1.f, 1.f, 2.f, 2.f});
+  tcd.addOutput(
+      std::vector<float>{1.f, 1.f, 1.f, 1.666666667, 1.666666667, 1.666666667, 2.f, 2.f, 2.f});
+  _context->setBackends({"cpu"});
+
+  SUCCEED();
+}
+
+TEST_F(GenModelTest, neg_OneOp_ResizeBilinear_InvalidSizeLen)
```
Comment (wateret) [inline tests/nnfw_api/src/one_op_tests/ResizeBilinear.cc:61]: 👍 for negative test cases!

**Review by wateret — APPROVED** [review]
LGTM

File: tests/nnfw_api/src/one_op_tests/ResizeBilinear.cc [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                        {{1.f, 1.f, 1.f, 1.666666667, 1.666666667, 1.666666667, 2.f, 2.f, 2.f}}));
+  _context->setBackends({"acl_cl", "acl_neon", "cpu"});
+
+  SUCCEED();
+}
+
+TEST_F(GenModelTest, OneOp_ResizeBilinear_SizeToVar)
+{
+  CircleGen cgen;
+  int size = cgen.addTensor({{2}, circle::TensorType::TensorType_INT32});
```
Comment (ragmani) [inline tests/nnfw_api/src/one_op_tests/ResizeBilinear.cc:44]: This line was fixed from `{1}` to `{2}` by force push

**Comment by hseok-oh** [comment]
```
22:02:35  [ RUN      ] GenModelTest.neg_OneOp_ResizeBilinear_InvalidSizeVal
22:02:35  nnfw_api_gtest: /opt/jenkins_slave/workspace/nnfw/master/pr-cross-runtime-test-pipeline@2/runtime/onert/core/src/util/ShapeInference.cc:484: onert::ir::Shape onert::shape_inference::inferResizeBilinearShape(const onert::ir::Shape&, int32_t, int32_t): Assertion `in_shape.rank() == 4' failed.
22:02:35  /jenkins/slave/workspace/nnfw/master/pr-cross-runtime-test-pipeline/Product/out/test/command/unittest: line 80:  9479 Aborted                 $TEST_BIN $(get_gtest_option)
22:02:35  Product/out/unittest_standalone/nnfw_api_gtest failed... return code: 134
```

Same issue with https://github.com/Samsung/ONE/pull/4127#issuecomment-688586129

**Comment by ragmani** [comment]
> Same issue with #4127 (comment)

The error was cause by incorrect shape setting in the testcase. The assertion `in_shape.rank() == 4` is right

**Review by hseok-oh — APPROVED** [review]
LGTM

**Review by hseok-oh — APPROVED** [review]

**Review by wateret — APPROVED** [review]
LGTM

**Pull request merged**